framework/include/samplers/Sampler.h

```diff
@@ -137,6 +137,21 @@ class Sampler : public MooseObject, public SetupInterface, public DistributionIn
    */
   dof_id_type getLocalRowEnd();
 
+  /**
+   * Return true if the sample data is distributed among all processors.
+   */
+  bool useDistributedSampleData() { return _use_distributed_sample_data; };
```

Comment:
I suggest changing this to:

```cpp
bool isSampleDataDistributed() const
```

Reply:
The name now, for me, implies that you are call a function to have the class us e distributed data

Reply:
I will change this.